### DISCUSSION TITLE
Select / Combobox / Listbox Options on top of Dialog

### DISCUSSION THREAD

**Discussion by adamleemiller** [tailwindlabs/headlessui#2759] [Help]
Does anyone know how to get the listbox options to be "on top" of the dialog / modal by chance? I am using Vue 3, if it makes a difference.

![Screenshot 2023-09-18 at 15 34 14](https://github.com/tailwindlabs/headlessui/assets/70930980/85af2b12-156f-49f4-b389-fda552eedc80)

**Comment by adamleemiller** [comment]
Not sure if it will cause problems anywhere else but I changed an "overflow-hidden" to "overflow-visible" and not it works. Feel free to ask me questions for anyone who has this same issue in the future.

**Reply by relativepathson** [reply]
@adamleemiller I encountered the same using a Listbox within a Dialog. I only added z-10 to the classnames of the ListboxOptions component and it solved the glitch.
